go/ct/rlz/conditions.go

```diff
@@ -724,7 +734,20 @@ func (c *isAddressCold) Restrict(generator *gen.StateGenerator) {
 }
 
 func (c *isAddressCold) GetTestValues() []TestValue {
-	return IsAddressWarm(c.key).GetTestValues()
+	property := Property(c.String())
```

Comment:
The "Property" targeted by the test value should be the same as for the `isAddressWarm` condition.

Reply:
The String() function new return the same string.

Reply:
That's not good (see comment for the `String` function). There is no need to link the `Property` to the string function of the condition. It can be something like
```
Property(fmt.Sprintf("warm(%v)", c.key))
```
here and for the complement constraint.